Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gradle:6.8.2-jdk11 as java_generator
+FROM gradle:6.6-jdk11 as java_generator
```

Comment:
Why we are downgrading version?

Reply:
I use the same version in gradle wrapper